backend/scandamus/game/tournament_match.py

```diff
@@ -262,6 +271,7 @@ def create_matches(tournament, players, round_number):
    #Match.objects.bulk_create(matches)
     with transaction.atomic():
         tournament.matches.add(*matches)
-        tournament.save()
+        tournament.save(update_fields=['bye_player', 'matches'])
```

Comment:
```suggestion
        tournament.save(update_fields=['bye_player'])
```
manytomanyのフィールドはupdate_fieldsに指定できない仕様
addで追加しているので問題ないようです

Reply:
ありがとうございます🙏こちらmergeしました